client/src/index.tsx

```diff
@@ -1,17 +1,36 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
 import * as serviceWorker from './serviceWorker';
+import App from './App';
+import { Drizzle } from '@drizzle/store';
+// import MyStringStore from './contracts/MyStringStore.json';
+
+const MyStringStore = require('./contracts/MyStringStore.json');
+
+// let drizzle know what contracts we want and how to access our test blockchain
+const options = {
+    contracts: [MyStringStore],
+    web3: {
+        fallback: {
+            type: "ws",
+            url: "ws://127.0.0.1:9545",
+        },
+    },
+};
+
+// setup drizzle
+const drizzle = new Drizzle(options);
```

Comment:
When I run the app this is complaining.

Reply:
`TypeError: options.isMergeableObject is not a function`

Reply:
hmmmm, looking at the types for `@drizzle/store` im not sure why this wouldnt work. Im gonna download and run myself